master.yaml

```diff
@@ -56,6 +61,8 @@ Resources:
                 VPC: !GetAtt VPC.Outputs.VPC
                 Subnets: !GetAtt VPC.Outputs.PublicSubnets
                 SecurityGroup: !GetAtt SecurityGroups.Outputs.LoadBalancerSecurityGroup
+                PublicAlbAcmCertificate:
+                    !Ref PublicAlbAcmCertificate
```

Comment:
This is where I would expect to see the actual `arn` that is currently in the `load-balancers.yaml` template. Also note that since no value for the `PublicAlbAcmCertificate` parameter, that "no value" value is being passed on to the load-balancers.yaml template and that is what is causing the Ssl and NoSsl conditions to be the way they are as described above.